features/step_definitions/openid_steps.rb

```diff
@@ -0,0 +1,64 @@
+When /^I register a new client$/ do
+  client_registration_url = "/openid_connect/clients"
+  post client_registration_url,
+       {
+         redirect_uris: ["http://localhost:3000"] # Not actually used
+       }
+end
+
+Given /^I send a post request from that client to the token endpoint using "([^\"]*)"'s credentials$/ do |username|
+  client_json = JSON.parse(last_response.body)
+  user = User.find_by(username: username)
+  token_endpoint_url = "/openid_connect/access_tokens"
+  post token_endpoint_url,
+    {
```

Comment:
Align the parameters of a method call if they span more than one line.<br>Redundant curly braces around a hash parameter.